utility/path.go

```diff
@@ -4,38 +4,12 @@ import (
 	"fmt"
 	"io/ioutil"
 	"path/filepath"
-	"strings"
 
 	"github.com/bitrise-io/go-utils/pathutil"
 )
 
-// FilterFunc ...
-type FilterFunc func(pth string) (bool, error)
-
-// FilterPaths ...
-func FilterPaths(paths []string, filters ...FilterFunc) ([]string, error) {
-	filtered := []string{}
-
-	for _, pth := range paths {
-		allowed := true
-		for _, filter := range filters {
-			if allows, err := filter(pth); err != nil {
-				return []string{}, err
-			} else if !allows {
-				allowed = false
-				break
-			}
-		}
-		if allowed {
-			filtered = append(filtered, pth)
-		}
-	}
-
-	return filtered, nil
-}
-
 // ListEntries ...
-func ListEntries(dir string, filters ...FilterFunc) ([]string, error) {
+func ListEntries(dir string, filters ...pathutil.FilterFunc) ([]string, error) {
```

Comment:
Should not it go to `go-utils/pathutil`? I feel like this function is generic enough and has no ios specific logic.

Reply:
The same applies to `EscapeGlobPath`, I suggest moving it to go-utils/pathutil